ArcGISRuntimeSDKQt_CppSamples/Maps/OpenMobileMap_MapPackage/OpenMobileMap_MapPackage.h

```diff
@@ -39,9 +42,16 @@ class OpenMobileMap_MapPackage : public QQuickItem
   void componentComplete() override;
   static void init();
 
+private slots:
+  void packageLoaded(const Esri::ArcGISRuntime::Error& e);
```

Comment:
optional, but I find it useful to name slots as starting with "on" or "handle" - so `handlePackageLoaded`

Reply:
Very well. This function is now removed, but I will keep it in mind for future reference.